README.md

```diff
@@ -1,100 +1,64 @@
-
-### Activating the environment
-After creating the `csmdev` environment and installing cmake and libcsm into it, we need to activate it. Right now, cmake, libcsm, and their dependencies are isolated to a conda environment and we need to tell conda that we want to use it. The activation command depends on your shell.
-* **bash**: `source activate csmdev`
-* **tcsh**: `conda activate csmdev`
-> *You can add these to the end of your $HOME/.bashrc or $HOME/.cshrc if you want the `csmdev` environment to be active in every new terminal.*
+Community Sensor Model (CSM) compliant sensor models created by USGS Astrogeology
+Science Center.
 
-## Building CSM-CameraModel
-After you've set up conda, you can build CSM-CameraModel:
-
-1. Fork `USGS-Astrogeology/CSM-CameraModel` if you don't already have a fork.
-2. Clone your fork of `CSM-CameraModel` *with `--recursive` option to get the gtest submodule*.
-```bash
-git clone --recursive git@github.com:<your-username>/CSM-CameraModel.git
-cd CSM-CameraModel
-git remote add upstream git@github.com:USGS-Astrogeology/CSM-CameraModel.git
-```
-3. Sync your fork with `upstream` and ensure the gtest submodule is init'd if your fork is old.
-```bash
-git pull upstream master
-git submodule update --init --recursive
-git push -u origin master
-```
-4. `mkdir build && cd build`
-5. `cmake .. && make`
-6. `ctest`
+CSM-CameraModel contains two different sensor models. The first, is a generic
+framing camera model written from scratch. The second is a generic line scan
+camera model based on code from BAE Systems Information and Electronic Systems
+Integration Inc.
+
+## Using CSM-CameraModel
+
+This library is a CSM plugin library that is intended to be dynamically loaded
+at run time along side the
+[CSM API library](https://github.com/USGS-Astrogeology/csm).
+
+Once, the library is loaded, it can be accessed through the CSM Plugin interface.
+For an example of how to do through the CSM c++ interface see the SensorModelFactory
+class in [SensorUtils](https://github.com/USGS-Astrogeology/SensorUtils).
+For an example of how to do this through the CSM Python bindings see this
+[notebook](http://nbviewer.jupyter.org/gist/thareUSGS/4c0eb72799edc33ff4816b2587027148).
+
+From the CSM Plugin interface, a generic framing camera model
+(USGS_ASTRO_FRAME_SENSOR_MODEL) or generic line scan camera model
+(USGS_ASTRO_LINE_SCANNER_SENSOR_MODEL) can be instantiated from suitable Image
+Support Data (ISD). Under the CSM standard, each plugin library can define its
+own ISD format. This library uses an auxiliary JSON formatted file that must be
+next to the image file passed to the CSM::ISD class. We provide an OpenAPI
+server for generating these,
+[pfeffernusse](https://github.com/USGS-Astrogeology/pfeffernusse). The swagger
+specification is located on
+[swaggerhub](https://app.swaggerhub.com/apis/USGS-Astro/pfeffernusse2/0.1.4).
```

Comment:
Should this be: https://app.swaggerhub.com/apis/USGS-Astro/pfeffernusse2/0.1.4-oas3 ?